test/integration/sample_test.go

```diff
@@ -0,0 +1,79 @@
+package samples_test
```

Comment:
nit: `tf_samples_test`?

Reply:
I think its clear from the repo name and since we don't have any other kinds of samples in repo.